app/experimenter/nimbus-ui/src/components/InputRadios/index.stories.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import InputRadios from ".";
+import { useCommonForm } from "../../hooks";
+
+export const exampleFieldNames = ["foo"] as const;
+const defaultValues = {
+  foo: "true",
+};
+
+const { FormErrors, formControlAttrs } = useCommonForm<
+  typeof exampleFieldNames[number]
+>(defaultValues, true, {}, () => {});
+
+export default {
+  title: "components/InputRadios",
+  component: InputRadios,
+};
+
+export const basic = (
+  <InputRadios
+    name="foo"
+    options={[
+      { label: "Yes", value: "true" },
+      { label: "No", value: "false" },
+    ]}
+    {...{ FormErrors, formControlAttrs }}
+  >
+    Howdy, neighbor
+  </InputRadios>
+);
```

Comment:
I think I'm missing something here as this story doesn't display, any ideas @lmorchard?

Reply:
Nevermind, I was being silly :)